app/assets/js/rtMedia.js

```diff
@@ -1252,15 +1252,13 @@ function get_parameter( parameter, data ) {
 function rtm_upload_terms_activity() {
 	// Check if upload term checkbox is there.
 	if ( jQuery( '#rtmedia_upload_terms_conditions' ).length > 0) {
-		// Handle on click event.
-		jQuery( '#whats-new' ).click( 'on', function () {
-			if ( false === jQuery('#rtmedia_upload_terms_conditions').prop('checked') ){
-				// By default prevent user to upload media.
-				jQuery( '#aw-whats-new-submit' ).attr( 'disabled', 'disabled' );
-				// On click of upload terms checkbox enable post button.
-				jQuery( '#rtmedia_upload_terms_conditions' ).click(function () {
-					jQuery( '#aw-whats-new-submit' ).attr( 'disabled', ! this.checked );
-				});
+		// Handle error on click event.
+		jQuery( '#bp-nouveau-activity-form' ).on( 'click', '#aw-whats-new-submit', function ( event ) {
+			if ( false === jQuery( '#rtmedia_upload_terms_conditions' ).prop( 'checked' ) && jQuery( '#whats-new-form #message' ).length === 0 ) {
+				event.preventDefault();
+				var selector = jQuery( '.rtmedia-upload-terms' );
+				var rt_alert_msg = ( ( typeof rtmedia_upload_terms_check_terms_message ) === 'string' ) ? rtmedia_upload_terms_check_terms_message : rtmedia_upload_terms_check_terms_default_message;
```

Comment:
Where will this message come from?

Reply:
It is being localized.

Reply:
in different PR?

Reply:
I have not added this. It was already localized in RTMediaUploadTerms.php I have just used over here.

Reply:
ok, cool